### ISSUE TITLE
nanoc co -W throws an error because src/layouts does not exist.

### ISSUE THREAD

**Issue by 0x1eef** [nanoc/nanoc#1625]
### Steps to reproduce

I have setup nanoc to use "src/" as the content directory. 
It is not using any sort of layout, although a directory is specificed in `nanoc.yml`.

```yaml
# nanoc.yml
data_sources:
  - type: filesystem
    encoding: UTF-8
    content_dir: src/
    layouts_dir: src/layouts
```

### Expected behavior

I would expect `nanoc co -W` to not throw an error, or print a warning that the layout directory doesn't exist.

### Actual behavior

```
Listening for site changes…
/usr/home/0x1eef/.gems/gems/listen-3.7.1/lib/listen/adapter/config.rb:16:in `realpath': No such file or directory @ rb_check_realpath_internal - /project/src/layouts (Errno::ENOENT)
......
```

### Details

(Describe other details, if any, that you believe might be relevant.)

### Crash log

(When reporting a crash, create a [Gist](https://gist.github.com/) with the contents of the `crash.log` file, and link the Gist here.)

**Comment by denisdefreyne** [comment]
Good catch! I’ve prepared a fix for this: #1626

**Comment by 0x1eef** [comment]
Thank you for the fix !
